scalableminds/webknossos
# Allow to remove fallback layer

This PR makes it possible to remove the fallback layer in a volume tracing. To do this a button in the layer settings header was added.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- start a volume or hybrid tracing with existing segmentation layer
- go to the dataset settings
- next to the segmentation layer's settings header should be a minus button
- on click this should open a confirm pop up
- when selecting ok the layers segmentation should be removed and the layer should be reloaded
- the minus button should disappear
- when doing a volume/hybrid tracing with a task the button should not be there

### Issues:
- fixes #4741

------
